### PR TITLE
New textures for u_inventory

### PR DESCRIPTION
**Pull request by ObaniGemini**
Minimalistic so lighter and easier to understand textures.
Also resized 128px textures to 64px and fixed some problems of https://github.com/MinetestForFun/unified_inventory/pull/1

**470kb -> 78kb**

I'll add complementary textures to special servers if it gets merged (soundset, graveyard, 3d_armor)

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -59,7 +59,7 @@ dofile(modpath.."/api.lua")
 dofile(modpath.."/internal.lua")
 dofile(modpath.."/callbacks.lua")
 dofile(modpath.."/register.lua")
-dofile(modpath.."/bags.lua")
+--dofile(modpath.."/bags.lua")
```
Comment (Lymkwi) [inline init.lua]: Why was this touched?

**Comment by Lymkwi** [comment]
I'm okay with all of this (except the little line of code). Could you just squash all your commits into one please?

File: init.lua (hunk 1 above)
Comment (ObaniGemini) [inline init.lua]: Oh.
RREDesign added it in his commit to remove bags from creative.
As I was on creative branch, it was added. Should I remove the whole commit ?

EDIT : removed

Comment (Lymkwi) [inline init.lua]: You should, yes.

**Comment by ObaniGemini** [comment]
Squashed. Added credits to the Readme.md file.

**Comment by ObaniGemini** [comment]
I think I have finished everything I had to do on that PR.
Some checks ?

**Comment by BetterToAutomateTheWorld** [comment]
Perfect, :+1:

**Comment by ObaniGemini** [comment]
Up, I need at least 4 or 5 approvals to merge this, please :)

**Comment by BetterToAutomateTheWorld** [comment]
I think you can merge it, many (surely all) people will enjoy your new textures :)

**Pull request merged**